### PR TITLE
Add default String and Float family

### PR DESCRIPTION
**Pull request by jq**
@deerzq @yinlou

### CODE REVIEW

File: core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java [hunk 1]
```diff
@@ -167,9 +167,18 @@ public void getFamilyEmpty() throws Exception {
     Pair<String, String> p = Features.getFamily("");
   }
 
-  @Test(expected = RuntimeException.class)
-  public void getFamilyNotLABEL() throws Exception {
+  @Test
+  public void getDefaultStringFamily() throws Exception {
+    Pair<String, String> p = Features.getFamily("LABE");
```
Comment (deerzq) [inline core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java]: LABEL?

File: core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java [hunk 2]
```diff
@@ -167,9 +167,18 @@ public void getFamilyEmpty() throws Exception {
...
+    assertEquals("", p.getLeft());
+    assertEquals(Features.DEFAULT_STRING_FAMILY, Features.getStringFamily(p));
+  }
+
+  @Test
+  public void getDefaultFloatFamily() throws Exception {
     Pair<String, String> p = Features.getFamily("LABE");
```
Comment (deerzq) [inline core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java]: LABEL?

Comment (jq) [inline core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java]: no, this is just feature name, it was to throw, I can change it to a better name

**Comment by deerzq** [comment]
LGTM

**Pull request merged**